src/qml/qmlapplication.cpp

```diff
@@ -47,7 +47,7 @@ QmlApplication::QmlApplication(
           m_fileWatcher({m_mainFilePath}) {
     m_pCoreServices->initialize(app);
     SoundDeviceStatus result = m_pCoreServices->getSoundManager()->setupDevices();
-    if (result != SOUNDDEVICE_OK) {
+    if (result != SoundDeviceStatus::Ok) {
         qCritical() << "Error setting up sound devices" << result;
         exit(result);
```

Comment:
```suggestion
        qCritical() << "Error setting up sound devices" << result;
        exit(static_cast<int>(result));
```

Reply:
I think you can fix the `no match for ‘operator<<’ (operand types are ‘QDebug’ and ‘SoundDeviceStatus’)` problem either by just also `static_cast`ing to an int or you can experiment with adding `Q_DECLARE_METATYPE(SoundDeviceStatus)` and/or `
Q_ENUM(SoundDeviceStatus)` in the same file as the enum definition.

Reply:
hmm...
For `exit` casting to int is fine.
For the debug output a string would be appreciated. I'm already looking into the latter.
`Q_DECLARE_METATYPE(SoundDeviceStatus);` in src/soundio/sounddevicestatus.h
+
`qDebug() << QVariant::fromValue(status).toString();`
compiles without warnings and works.
Though, is this the preferred solution?

Reply:
I was wrong, to string conversion is tricky / elaborate and I'm not in the mood to crack that nut.
I'll got static_cast<int> for both the debug string and the exit code.

Please double-check, then I'll squash the fixup.